src/org/opendatakit/briefcase/ui/settings/SettingsPanel.java

```diff
@@ -63,7 +63,7 @@ public SettingsPanel(SettingsPanelForm form, BriefcasePreferences appPreferences
     form.onHttpProxy(appPreferences::setHttpProxy, appPreferences::unsetHttpProxy);
     form.onReloadCache(() -> {
       formCache.update();
-      JOptionPane.showMessageDialog(getContainer(), "Forms successfully reloaded from storage location.");
+      JOptionPane.showMessageDialog(getContainer(), "Forms successfully reloaded from storage location.", "", JOptionPane.PLAIN_MESSAGE);
```

Comment:
Did we consider using APP_NAME as the third argument instead of ""?

Reply:
Currently, <code>APP_NAME</code> is defined in <code>org.opendatakit.briefcase.ui.MainBriefcaseWindow</code>. In order to use it in <code>SettingsPanel</code> I would need to define it in one of the constants file so both the filed could use it. Shoud I go ahead and do that?

Reply:
I’d say make APP_NAME public in MainBriefcaseWindow, and statically import it into SettingsPanel. Unless @ggalmazor prefers something else.

Reply:
@ggalmazor Do you agree with this?

Reply:
@dcbriccetti Is this common? It seems odd to me for a dialog to have the name of the app but I have to admit I'm not sure what other apps do in this case. I would expect the title to be something like "Success"

Reply:
@lognaturel, thanks for jumping in. I may have been wrong about this. My ancient Swing knowledge is mixing in with my ancient OS/2 Presentation Manager knowledge.

Two of the three other places in the code where PLAIN_MESSAGE is used have the application title, and the other has "".

Reply:
😊The two that use the application title are the introductory dialogs so that makes a lot of sense to me. I'd still prefer "Success" but don't feel too strongly about it.

@kunalmulwani, thanks for looking into this issue and making the change. @ggalmazor can be the tie breaker on this one. ;)